extension.json

```diff
@@ -0,0 +1,344 @@
+			]
+		},
+		"ext.maps.gm3.markerwithlabel": {
+			"group": "ext.maps",
+			"targets": [
+				"mobile",
+				"desktop"
+			],
+			"scripts": [
+				"includes/services/GoogleMaps3/gm3-util-library/markerwithlabel.js"
+			],
+			"styles": [
+				"includes/services/GoogleMaps3/gm3-util-library/markerwithlabel.css"
+			]
+		},
+		"ext.maps.gm3.geoxml": {
+			"group": "ext.maps",
+			"targets": [
+				"mobile",
+				"desktop"
+			],
+			"scripts": [
+				"includes/services/GoogleMaps3/geoxml3/geoxml3.js",
+				"includes/services/GoogleMaps3/geoxml3/ZipFile.complete.js",
+				"includes/services/GoogleMaps3/geoxml3/ProjectedOverlay.js"
+			]
+		},
+		"ext.maps.gm3.earth": {
+			"group": "ext.maps",
+			"targets": [
+				"mobile",
+				"desktop"
+			],
+			"scripts": [
+				"includes/services/GoogleMaps3/googleearth-compiled.js"
+			]
+		},
+		"ext.maps.openlayers": {
+			"dependencies": [
+				"ext.maps.common"
+			],
+			"group": "ext.maps",
+			"targets": [
+				"mobile",
+				"desktop"
+			],
+			"scripts": [
+				"includes/services/OpenLayers/OpenLayers/OpenLayers.js",
+				"includes/services/OpenLayers/OSM/OpenStreetMap.js",
+				"includes/services/OpenLayers/jquery.openlayers.js",
+				"includes/services/OpenLayers/ext.maps.openlayers.js"
+			],
+			"styles": [
+				"includes/services/OpenLayers/OpenLayers/theme/default/style.css"
+			],
+			"messages": [
+				"maps-markers",
+				"maps-copycoords-prompt",
+				"maps-searchmarkers-text"
+			]
+		},
+		"ext.maps.leaflet": {
+			"dependencies": [
+				"ext.maps.common"
+			],
+			"group": "ext.maps",
+			"targets": [
+				"mobile",
+				"desktop"
+			],
+			"scripts": [
+				"includes/services/Leaflet/jquery.leaflet.js",
+				"includes/services/Leaflet/ext.maps.leaflet.js"
+			],
+			"messages": [
+				"maps-markers",
+				"maps-copycoords-prompt",
+				"maps-searchmarkers-text"
+			]
+		}
+	},
+	"ResourceFileModulePaths": {
+		"localBasePath": "",
+		"remoteExtPath": "Maps"
+	},
+	"Hooks": {
+		"AdminLinks": [
+			"MapsHooks::addToAdminLinks"
+		],
+		"ArticleFromTitle": [
+			"MapsHooks::onArticleFromTitle"
+		],
+		"MakeGlobalVariablesScript": [
+			"MapsHooks::onMakeGlobalVariablesScript"
+		],
+		"CanonicalNamespaces": [
+			"MapsHooks::onCanonicalNamespaces"
+		],
+		"LoadExtensionSchemaUpdates": [
+			"MapsHooks::onLoadExtensionSchemaUpdates"
+		],
+		"ArticlePurge": [
+			"MapsHooks::onArticlePurge"
+		],
+		"LinksUpdateConstructed": [
+			"MapsHooks::onLinksUpdateConstructed"
+		],
+		"ParserAfterTidy": [
+			"MapsHooks::onParserAfterTidy"
+		],
+		"ParserClearState": [
+			"MapsHooks::onParserClearState"
+		],
+		"ParserFirstCallInit": [
+			"MapsHooks::onParserFirstCallInit1",
+			"MapsHooks::onParserFirstCallInit2",
+			"MapsHooks::onParserFirstCallInit3",
+			"MapsHooks::onParserFirstCallInit4",
+			"MapsHooks::onParserFirstCallInit5",
+			"MapsHooks::onParserFirstCallInit6",
+			"MapsHooks::onParserFirstCallInit7"
+		],
```

Comment:
Why not have this only in `Maps.php`? I don't see any advantage of having it in the JSON file